aip/general/0130.md

```diff
@@ -0,0 +1,77 @@
+---
+id: 130
+state: draft
+created: 2023-03-313
+updated: 2023-03-313
+placement:
+  category: operations
+  order: 9
+---
+
+# Methods
+
+Methods are conceptually synonymous with
```

Comment:
I know this is just a draft but...can you finish the sentence? The suspense is killing me!

Reply:
sorry! should have reviewed earlier.